TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,22 @@
 
 ## Unreleased
 
+### Fix varation bug with Products reports #6647
+
+
+1. Add two variable products. You want to have at least one variable for each product. 
+
+Product A - color:black
+Product A - color:white
+
+Prorudct B - size:small
+Product B - size:medium
+
+2. Make an order for each product.
+3. Navigate to Analytics -> Products
```

Comment:
During my test, I had to reimport historical data in order to see the data reflected in the report. I think it's worth considering adding a reference on how to do this in case if tester bumps into it as well: https://docs.woocommerce.com/document/woocommerce-analytics/#analytics-settings__import-historical-data

Reply:
Thank you @ilyasfoo 👍 good suggestion. I will add it before merging the PR.